### PR TITLE
[Doc-enhance][updated-doc][1827919]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Added function details in Authentication namespace.

### CODE REVIEW

File: msteams-platform/tabs/how-to/using-teams-client-sdk.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ See the following tables to understand commonly used SDK functions. The [SDK ref
...
 |`microsoftTeams.authentication.notifySuccess(result?: string, callbackUrl?: string)`|Notifies the frame that initiated the authentication request that the request was successful and closes the authentication window|[function](/javascript/api/@microsoft/teams-js/microsoftteams.authentication?view=msteams-client-js-latest&preserve-view=true)|
 |`microsoftTeams.authentication.notifyFailure(reason?: string, callbackUrl?: string)`|Notifies the frame that initiated the authentication request that the request failed and closes the authentication window.|[function](/javascript/api/@microsoft/teams-js/microsoftteams.authentication?view=msteams-client-js-latest&preserve-view=true)|
+|`microsoftTeams.authentication.getAuthToken(authTokenRequest: AuthTokenRequest)`|Requests an Azure AD token to be issued on behalf of the app. The token is acquired from the cache if it is not expired. Otherwise, a request is sent to Azure AD to obtain a new token.|[function](/javascript/api/@microsoft/teams-js/microsoftteams.authentication?view=msteams-client-js-latest#getAuthToken_AuthTokenRequest_&preserve-view=true)|
```
Comment (v-abirade) [inline msteams-platform/tabs/how-to/using-teams-client-sdk.md:55]: Send request to issue Azure AD token on behalf of the app. (Please use active voice)
The token can be acquired from the cache, if it has not expired. (sentence correction)

**Review by v-abirade — CHANGES_REQUESTED** [review]
Changes requested

File: msteams-platform/tabs/how-to/using-teams-client-sdk.md (hunk 1 above)
Comment (Mahesh2516) [reply msteams-platform/tabs/how-to/using-teams-client-sdk.md]: accepted suggestion

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-abirade — APPROVED** [review]
Approve

**Pull request merged**